develop-env/conda/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM centos:7
+ARG PYTHON_VERSION=3.7m
+ARG CONDA_VERSION=4.5.4
+USER root
+RUN yum install -y mlocate
+RUN adduser -u 1004 -m -s /bin/bash saluser
+RUN yum install -y gcc-c++ make ncurses-libs xterm xorg-x11-fonts-misc java-1.7.0-openjdk-devel maven python-devel swig git tk-devel wget bzip2
+COPY lsst-ts.repo /tmp/lsst-ts.repo
+RUN cat /tmp/lsst-ts.repo >> /etc/yum.conf && yum install -y OpenSpliceDDS-6.9.0-1
+USER saluser
+WORKDIR /home/saluser
+RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh && chmod +x miniconda.sh && ./miniconda.sh -b
+RUN source ~/miniconda3/bin/activate && conda install -c lsstts python=3.7 ts-salobj
+ARG BRANCH=v3.10.0
+RUN mkdir repos && cd repos && git clone https://github.com/lsst-ts/ts_sal \ 
+    && cd ts_sal && git checkout ${BRANCH}
+ARG BRANCH=v3.10.2
```

Comment:
Use different ARG name so we can actually parameterize this. 

e.g.; https://github.com/lsst-ts/ts_Dockerfiles/blob/a3ac2bfbb6c92fbeb406226a3c8a3ba68e1f2f07/develop-env/salobj_4/Dockerfile#L17-L21

```
ARG sal_v="tags/v3.10.0"
ARG xml_v="tags/v3.10.0"
```